DashAI/__init__.py

```diff
@@ -7,29 +7,19 @@
 
 
 def run():
-    url = "http://localhost:3000/"
+    url = "http://localhost:8000/app"
 
-    procs = [
-        Popen(
+    p = Popen(
             "python -m DashAI.back.main",
             shell=True,
```

Comment:
If two processes are no longer running, then there is no need to launch separate processes.
Here you would launch the FastAPI server directly.
```suggestion
from DashAI.back.main import app

if __name__ == "__main__":
    uvicorn.run(app, host="127.0.0.1", port=8000)
```

And delete everything else.

Reply:
Done, i also added the auto pop up from the browser after 1s